controllers/workspace/devworkspace_controller.go

```diff
@@ -240,6 +241,22 @@ func (r *DevWorkspaceReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		HttpClient:         httpClient,
 	}
 
+	// TODO: Use merge code rather than deepCopy which overwrites?
+	if len(workspace.Spec.Template.Components) == 0 && config.Workspace.DefaultTemplate != nil {
+		var projectsCopy []v1alpha2.Project
+		if len(workspace.Spec.Template.Projects) > 0 {
+			projectsCopy = make([]v1alpha2.Project, len(workspace.Spec.Template.Projects))
+		}
+
+		// TODO: Use deepCopyInto instead?
+		defaultTemplateCopy := config.Workspace.DefaultTemplate.DeepCopy()
+		workspace.Spec.Template.DevWorkspaceTemplateSpecContent = *defaultTemplateCopy
+
+		if len(projectsCopy) > 0 {
+			workspace.Spec.Template.Projects = projectsCopy
```

Comment:
Rather than overwriting the Projects (which may be coming from the default template), we should append the existing projects